config/src/config/api_config.rs

```diff
@@ -116,6 +118,7 @@ impl Default for ApiConfig {
             max_runtime_workers: None,
             runtime_worker_multiplier: 2,
             gas_estimation: GasEstimationConfig::default(),
+            periodic_gas_estimation_ms: Some(30_000),
```

Comment:
do we want this on validators/ vfns?

Reply:
It only takes effect if the API endpoint is `enabled` in this config. So it won't take effect for prod VN/VFNs.

Reply:
Perfect